packages/mui-system/src/cssVars/createCssVarsProvider.test.js

```diff
@@ -33,9 +34,15 @@ describe('createCssVarsProvider', () => {
     // clear the localstorage
     storage = {};
     window.matchMedia = createMatchMedia(false);
+
+    // Currently supported Firefox does not support `color-scheme`.
+    // Instead of skipping relevant tests entirely we assert that they work differently in Firefox.
+    // This ensures that we're automatically notified once we remove older Firefox versions from the support matrix.
+    shouldSupportColorScheme = !/Firefox/.test(navigator.userAgent);
   });
   afterEach(() => {
     window.matchMedia = originalMatchmedia;
+    document.documentElement.style.setProperty('color-scheme', 'normal');
```

Comment:
I'll revert these changes since the component is responsible for that. Tests should only cleanup what they cause.

Reply:
Ah, I see. Yep, sure.